### PR TITLE
Move qss file from napari to plugin manager

### PR DESCRIPTION
**Pull request by goanpeca**
History was preserved on the original file, but we removed all qss not related to the plugin manager.

### CODE REVIEW

File: napari_plugin_manager/qt_plugin_dialog.py [hunk 1]
```diff
@@ -965,6 +969,9 @@ def setup_ui(self):
 
         self.packages_filter.setFocus()
 
+        stylesheet = get_current_stylesheet([STYLES_PATH])
+        self.setStyleSheet(stylesheet)
```
Comment (jaimergp) [inline napari_plugin_manager/qt_plugin_dialog.py:973]: Why is this need twice? It's a bit weird, but if that's how it is, so be it 🤷 Let's write a comment at least.

File: napari_plugin_manager/styles.qss [hunk 2]
```diff
@@ -0,0 +1,334 @@
+/* ------------ Plugin Dialog ------------ */
```
Comment (jaimergp) [inline napari_plugin_manager/styles.qss:1]: Let's put a little comment here that this must be loaded with `get_current_stylesheet`.

File: napari_plugin_manager/qt_plugin_dialog.py (hunk 1 above)
Comment (goanpeca) [reply napari_plugin_manager/qt_plugin_dialog.py:973]: Indeed que can only use the one in setupui, removing the one on init

File: napari_plugin_manager/styles.qss (hunk 2 above)
Comment (goanpeca) [reply napari_plugin_manager/styles.qss:1]: Sounds good

**Review by jaimergp — APPROVED** [review]

**Pull request merged**